WirelessSystem.cs

```diff
@@ -12,7 +12,7 @@ public class WirelessSystem : ModSystem
 	{
 		public static Dictionary<Point16, Point16> Links;
 		
-		public override void OnWorldLoad()
+		public override void OnModLoad()
```

Comment:
Does the Links dictionary get reset when you leave and enter another world?

Reply:
The new commit should solve that.